svthalia/Reaxit
# Add a scrollbar

### Summary
Adds a scrollbar to the screens where that is suitable.

### How to test
Steps to test the changes you made:
1. Scroll on screens where that is possible, and look at the scrollbar on the right.
2. See if there's any screens I forgot.
